src/test/java/io/openliberty/tools/intellij/it/TestUtils.java

```diff
@@ -388,13 +388,12 @@ public static void validateTestReportExists(Path pathToTestReport) {
                     Thread.sleep(retryIntervalSecs * 1000);
                 } catch (Exception e) {
                     e.printStackTrace(System.out);
-                    continue;
                 }
-                continue;
+            } else {
+                return;
             }
-
-            return;
         }
+        throw new IllegalStateException("Timed out waiting for test report: " + pathToTestReport + " file to be created.");
```

Comment:
how do we further handled the exception

Reply:
When I investigated, I found that it's not always necessary to handle exceptions using a **_try-catch_** block for unchecked exceptions like **_IllegalStateException_**.


In this case, we don't need to **catch** the exception because we've checked the condition 100 times (and the **fileExists** value remains `false` throughout). Therefore, we can `throw` the exception directly without implementing further catch mechanisms for the corresponding test, resulting in **test failure**.